runtime-modules/referendum/src/tests.rs

```diff
@@ -795,6 +795,297 @@ fn winners_multiple_winners() {
+        let origin_voter2 = OriginType::Signed(account_id2);
+        let origin_voter3 = OriginType::Signed(account_id3);
+        let origin_voter4 = OriginType::Signed(account_id4);
+        let origin_voter5 = OriginType::Signed(account_id5);
+        let cycle_id = 1;
+        let winning_target_count = 2;
+
+        let option_to_vote_for1 = 0;
+        let option_to_vote_for2 = 1;
+        let option_to_vote_for3 = 2;
+        let stake = <Runtime as Trait>::MinimumStake::get();
+        let (commitment1, salt1) =
+            MockUtils::calculate_commitment(&account_id1, &option_to_vote_for1, &cycle_id);
+        let (commitment2, salt2) =
+            MockUtils::calculate_commitment(&account_id2, &option_to_vote_for2, &cycle_id);
+        let (commitment3, salt3) =
+            MockUtils::calculate_commitment(&account_id3, &option_to_vote_for3, &cycle_id);
+        let (commitment4, salt4) =
+            MockUtils::calculate_commitment(&account_id4, &option_to_vote_for3, &cycle_id);
+        let (commitment5, salt5) =
+            MockUtils::calculate_commitment(&account_id5, &option_to_vote_for3, &cycle_id);
+
+        Mocks::start_referendum_extrinsic(
+            origin.clone(),
+            winning_target_count.clone(),
+            cycle_id,
+            Ok(()),
+        );
+        Mocks::vote(
+            origin_voter1.clone(),
+            account_id1,
+            commitment1,
+            (stake * 3) + 1,
+            cycle_id.clone(),
+            Ok(()),
+        );
+        Mocks::vote(
+            origin_voter2.clone(),
+            account_id2,
+            commitment2,
+            stake * 2,
+            cycle_id.clone(),
+            Ok(()),
+        );
+        Mocks::vote(
+            origin_voter3.clone(),
+            account_id3,
+            commitment3,
+            stake,
+            cycle_id.clone(),
+            Ok(()),
+        );
+        Mocks::vote(
+            origin_voter4.clone(),
+            account_id4,
+            commitment4,
+            stake,
+            cycle_id.clone(),
+            Ok(()),
+        );
+        Mocks::vote(
+            origin_voter5.clone(),
+            account_id5,
+            commitment5,
+            stake,
+            cycle_id.clone(),
+            Ok(()),
+        );
+        // voting period starts at block 1
+        MockUtils::move_to_block(voting_stage_duration + 1);
+
+        Mocks::check_voting_finished(winning_target_count, cycle_id);
+
+        Mocks::reveal_vote(
+            origin_voter1.clone(),
+            account_id1,
+            salt1,
+            option_to_vote_for1,
+            Ok(()),
+        );
+        Mocks::reveal_vote(
+            origin_voter2.clone(),
+            account_id2,
+            salt2,
+            option_to_vote_for2,
+            Ok(()),
+        ); // at this time, the size of current_winners is 2, equal to winning_target_count, so revelaling smaller votes don't help the user3 to crowd out user2
```

Comment:
Doesn't this comment still describe the conditions as they were before the fix?
```suggestion
        );
```